src/coprocessor/dag/expr/builtin_encryption.rs

```diff
@@ -31,6 +34,118 @@ const SHA224: i64 = 224;
 const SHA256: i64 = 256;
 const SHA384: i64 = 384;
 const SHA512: i64 = 512;
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum AESMode {
```

Comment:
s/AESMode/AesMode/

Reply:
Aes is an abbreviation, I think abbreviations are more appropriate in uppercase letters.